### DISCUSSION TITLE
If I go to get ctklabel via mouse events, using event.widget.configure(bg_color) shows that there is no such attribute

### DISCUSSION THREAD

**Discussion by maxwangwyf** [TomSchimansky/CustomTkinter#2557] [Q&A]
def label_hover(event, text):
    bbox = event.widget.bbox("insert")
    if bbox is None:
        return 
    event.widget.configure(bg_color="#DDDDDD")
    def reset(event):
        event.widget.configure(bg_color='#FDFDFD')

    event.widget.bind("<Leave>", reset)

**Comment by Akascape** [comment]
@maxwangwyf You need to use `.winfo_parent()` to access the ctklabel class, as clicking on the ctklabel returns its internal subwidgets. But note that the returned value will be in string format, so you have to use `root.nametowidget` method in order to configure the ctklabel widget.

Example:
```python
import customtkinter

root = customtkinter.CTk()

def get_widget(event):
    widget = root.nametowidget(event.widget.winfo_parent()) #get the clicked widget class
    widget.configure(bg_color="red")
    
root.bind("<1>", get_widget) # bind the mouse click

label = customtkinter.CTkLabel(root, text="This is a Label")
label.pack(padx=10, pady=10)

root.mainloop()
```
